src/view/com/util/LoadingPlaceholder.tsx

```diff
@@ -42,58 +43,57 @@ export function PostLoadingPlaceholder({
 }: {
   style?: StyleProp<ViewStyle>
 }) {
-  const theme = useTheme()
-  const pal = usePalette('default')
+  const tokens = useTokens()
   return (
-    <View style={[styles.post, pal.view, style]}>
-      <LoadingPlaceholder
+    <Box bg='l0' row px='m' pt='xl' pb='xs'>
+      <Box
+        bg='l1'
         width={52}
         height={52}
-        style={[
-          styles.avatar,
-          {
-            position: 'relative',
-            top: -6,
-          },
-        ]}
+        radius='round'
+        mr='m'
+        ml='s'
+        top={-6}
       />
-      <View style={[s.flex1]}>
-        <LoadingPlaceholder width={100} height={6} style={{marginBottom: 10}} />
-        <LoadingPlaceholder width="95%" height={6} style={{marginBottom: 8}} />
-        <LoadingPlaceholder width="95%" height={6} style={{marginBottom: 8}} />
-        <LoadingPlaceholder width="80%" height={6} style={{marginBottom: 15}} />
-        <View style={s.flexRow}>
-          <View style={s.flex1}>
+      <Box flex={1}>
+        <Box gap='s'>
+          <Box bg='l1' width={100} height={6} radius='s' />
+          <Box bg='l1' width="95%" height={6} radius='s' />
+          <Box bg='l1' width="95%" height={6} radius='s' />
+          <Box bg='l1' width="80%" height={6} radius='s' mb='s' />
+        </Box>
+        <Box row>
+          <Box column>
```

Comment:
I replaced all this with Box just to show how easy it is to make basic components